fw-child/resources/js/child-functions.js

```diff
@@ -708,6 +709,25 @@ const $ = jQuery;
 
     };
 
+    //
+    // Dynamic logo block display scroll behavior
+    //
+
+    if ( $( '#floating-header' ).length && $( '#hero' ).length ) {
+      $( window ).on( 'scroll', function() {
+        const floatingHeaderBottom = $('#floating-header').offset().top + $('#floating-header').outerHeight();
```

Comment:
Adding spacing in parenthesis like on line 716 is a good idea on the new code. It complies with the WCS. Can you change it in this new code block at least for consistency within your code block? Note that we won't refactor the whole file for now. Just apply WCS to your new code.